test/util/sh.js

```diff
@@ -0,0 +1,26 @@
+import childProcess from 'node:child_process';
+
+const getCommandAndArgs = input => {
+  //"git checkout master".split(' ').slice(1)
+  if (!input || (input && typeof input !== 'string'))
+    throw new Error(`No command given to getCommandAndArgs. Please check input and/or type`);
+
+  const command = input.split(' ').reverse().pop();
```

Comment:
Splitting the command by space (same for `args` below) is maybe good enough if we use it only in our own tests. Is this something shelljs used an actual parser for?

Reply:
`shelljs` does alot of file i/o just for the `execSync` parth that is overkill imho, and use `execArgs` via a `paramsFile`: https://github.com/shelljs/shelljs/blob/8a960da8383666024bfdb2eff709a4fdf66333ee/src/exec.js#L100

might need a separate test yeah. will add

Reply:
Doh, looked at it wrongly. writing test for a test util? *facepalm*. Have amended the code